guides/how-we-do-user-research.md

```diff
@@ -239,9 +238,9 @@ recordings, or deleting data from prototypes.
 We have template discussion guides for three different kinds of research
 activities:
 
-- [Interviews and contextual research](https://drive.google.com/open?id=1XeBhmQL0jqFFsl2Cw-DOKVpabTRILGl-TjeyUDW90vM)
-- [Usability tests](https://drive.google.com/open?id=1hcVkLGJVDB9ZOoMIz93erUpk1RUgXN_ZcZflJxsFpHw)
-- [Workshops](https://drive.google.com/open?id=1ZHZwLBnUlDsxrvv1qj_ZpFU9QGsP_dIDLMVcGbEYswY)
+- [Interviews and contextual research](https://drive.google.com/open#1XeBhmQL0jqFFsl2Cw-DOKVpabTRILGl-TjeyUDW90vM)
```

Comment:
Ah, this looks like a regexp catching Google Drive links in its net, here and everywhere in this file.

Reply:
Whoops. I'll fix. Thanks.